vars/common.yml

```diff
@@ -25,6 +25,7 @@ supplier_frontend:
   instance_type: t2.micro
   min_instance_count: 1
   max_instance_count: 2
+  password_salt: ResetPasswordSalt
```

Comment:
OK maybe I'll do that - i.e. move it into app config file. From what it said in the docs you linked to I don't think it needs to be secret.